iam/api-client/src/main/java/com/google/iam/snippets/CreateServiceAccount.java

```diff
@@ -0,0 +1,74 @@
+/* Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.iam.snippets;
+
+// [START iam_create_service_account]
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.iam.v1.Iam;
+import com.google.api.services.iam.v1.IamScopes;
+import com.google.api.services.iam.v1.model.CreateServiceAccountRequest;
+import com.google.api.services.iam.v1.model.ServiceAccount;
+import java.util.Collections;
+
+public class CreateServiceAccount {
+
+  private static Iam service = null;
+
+  // Creates a service account.
+  public static void createServiceAccount(String projectId) throws Exception {
+    // String projectId = "my-project-id";
+
+    try {
+      initService();
+    } catch (Exception e) {
+      System.out.println("Failed to build service: \n" + e.toString());
+    }
+    ServiceAccount serviceAccount = new ServiceAccount();
+    serviceAccount.setDisplayName("DISPLAY_NAME");
```

Comment:
nit: It's nice to signal to the user these are configurable, and not required to be the literal value. 
```suggestion
    serviceAccount.setDisplayName("your-display-name");
```

Reply:
Done